_posts/2017/2017-11-09-devoxx-belguim-day-2.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "devoxx Belgium 2017 - Day 2"
+date: "2017-11-10 00:00:00"
+image: blog-header/devoxx-be-2017-2.jpg
+categories: events 
+authors: ["Dirk J."]
```

Comment:
Just Dirk, please :)

Reply:
Still the same question like yesterday: why? (and why not full names at all?)